app/ui/src/Pages/Project/pages/KG/components/Filters/components/TopicFilter/components/TopicList/TopicList.module.scss

```
@@ -2,33 +2,84 @@
 @import 'Styles/mixins';
 @import 'Styles/colors';
 
+.titlesContainer {
+  display: flex;
+  align-items: center;
+
+  margin-top: $grid-unit * 2;
+  margin-left: $grid-unit * 4;
+
+  .title {
+    @include font-caption
+    color: font-color(regular);
+    flex-grow: 1;
+  }
+
+  .paperCount {
+    @include font-caption
+    color: font-color(regular);
+  }
+
+  .relevance {
+    @include font-caption
+    color: font-color(regular);
+    margin-left: $grid-unit * 2;
```

You should use the same ordering when performing this type of operations, it is easier for the developer to read `X * $grid-unit` always instead of using `X * $grid-unit` some times and `$grid-unit * X` other times.

```suggestion
    margin-left: 2 * $grid-unit;
```